packages/react-dev-utils/openBrowser.js

```diff
@@ -120,8 +120,6 @@ function startBrowserProcess(browser, url, args) {
   try {
     var options = {
       app: { name: browser, arguments: args },
-      wait: false,
```

Comment:
Did you mean to delete the `wait` property?

Reply:
Yes! I noticed the default is set to false, as described [here](https://github.com/sindresorhus/open/tree/v8.2.0#wait)

Reply:
cool